ngraph/frontend/paddlepaddle/src/op/shape.cpp

```diff
@@ -0,0 +1,26 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "shape.hpp"
+#include <ngraph/opsets/opset6.hpp>
+
+namespace ngraph
+{
+    namespace frontend
+    {
+        namespace pdpd
+        {
+            namespace op
+            {
+                NamedOutputs shape(const NodeContext& node)
+                {
+                    auto data = node.get_ng_input("Input");
+                    return node.default_single_output_mapping(
+                        {std::make_shared<ngraph::opset6::ShapeOf>(data)}, {"Out"});
```

Comment:
Output type does matter. Spec says it should be int32 -- please set it as default is int64
```
AddOutput(
        "Out",
        "(LoDTensor), The shape of input tensor, the data type of the shape"
        " is int32_t, will be on the same device with the input Tensor.");
```